README.md

```diff
@@ -93,11 +118,11 @@ CKAN instance is running in `http://ckan:5000`. Login with one of the existing K
 >        root@1234567890ab:/# apache2ctl graceful
 >        root@1234567890ab:/# exit
 
-After that, the logged user should be able to create a new dataset:
+After that, the logged in user should be able to create a new dataset:
 
 ![Create a dataset in CKAN](tutorial-images/create-dataset.png)
 
-### 5. CKAN and WireCloud connections
+## 4. CKAN and WireCloud connections
```

Comment:
If the pasted order is correct, the numbering was interchanged. But I don't know if maybe the numbering was correct and the order in the readme was in fact incorrect?